mailer/models.py

```diff
@@ -43,21 +44,13 @@ def retry_deferred(self, new_priority=2):
 def email_to_db(email):
     # pickle.dumps returns essentially binary data which we need to encode
     # to store in a unicode field.
-    return base64.encodestring(pickle.dumps(email))
+    return force_text(base64.encodestring(pickle.dumps(email)))
```

Comment:
The changes in `email_to_db` and `db_to_email` were the big changes for this PR. Everything else is your basic python3/django2 upgrade stuff. What I think is happening is this: In django2, when writing a byte string to a DB field, it gets forced to text in a way that is making the `b` part of the content. So `b'foo'` would become `"b'foo'"`. Which is super not correct. So, we are forcing it to text here. So that doesn't happen.